packages/main/src/ColorPicker.ts

```diff
@@ -366,41 +366,43 @@ class ColorPicker extends UI5Element implements IFormInputElement {
 	}
 
 	_setMainColor(hueValue: number) {
-		if (hueValue <= 255) {
+		const hueValueMod = hueValue * 4.251;
```

Comment:
Usually, assigning to a new variable (`hueValueMod`) and using it throughout the function, is necessary when you need both the passed param (`hueValue`) and the modified one (`hueValueMod`) at the same time. But if you'll only use the modified one, the best would be to just have: `hueValue = hueValue * 4.251;` and continue using `hueValue` in the function as before.